src/box/lua/schema.lua

```diff
@@ -2383,46 +2383,62 @@ end
 local function privilege_resolve(privilege)
     local numeric = 0
     if type(privilege) == 'string' then
-        privilege = string.lower(privilege)
-        if string.find(privilege, 'read') then
+        local privilege_cp = privilege:lower():gsub('[, ]', '')
+        if privilege_cp:find('read') then
```

Comment:
I think we should refactor this code. It should honestly parse the line and find token. It is strange that privilege 'readwritexecute' did work and 'r e a d w r i t e e x e c u t e' will work.

Reply:
Refactored this code: extracted parsing of the privileges string into privilege_parse and now process it through a generic FSA.